<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">QtArc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>288</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExtract"/>
    <addaction name="actionTest"/>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionRename"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionDeselect"/>
    <addaction name="separator"/>
    <addaction name="actionAddFile"/>
    <addaction name="actionAddDirectory"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionExtract"/>
   <addaction name="actionAddFile"/>
   <addaction name="actionAddDirectory"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/document-new.png</normaloff>:/icons/icons/24x24/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/document-open.png</normaloff>:/icons/icons/24x24/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/document-save-as.png</normaloff>:/icons/icons/24x24/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionExtract">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/extract-archive.png</normaloff>:/icons/icons/24x24/extract-archive.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Extract</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+E</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>&amp;Test</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+T</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/document-properties.png</normaloff>:/icons/icons/24x24/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Properties</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionAddFile">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/add-files-to-archive.png</normaloff>:/icons/icons/24x24/add-files-to-archive.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;File</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+F</string>
   </property>
  </action>
  <action name="actionAddDirectory">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/add-folder-to-archive.png</normaloff>:/icons/icons/24x24/add-folder-to-archive.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;Directory</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+D</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/edit-cut.png</normaloff>:/icons/icons/24x24/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/edit-copy.png</normaloff>:/icons/icons/24x24/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/edit-paste.png</normaloff>:/icons/icons/24x24/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+V</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string notr="true">F2</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/edit-delete.png</normaloff>:/icons/icons/24x24/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string notr="true">Del</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../qtarc.qrc">
     <normaloff>:/icons/icons/24x24/help-about.png</normaloff>:/icons/icons/24x24/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>AboutQt</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+A</string>
   </property>
  </action>
  <action name="actionDeselect">
   <property name="text">
    <string>Deselect</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Shift+A</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qtarc.qrc"/>
 </resources>
 <connections/>
</ui>
